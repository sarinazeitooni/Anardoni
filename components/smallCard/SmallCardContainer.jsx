import React from "react";
import style from './scss/small-card.module.scss'
import SmallCardImage from './SmallCardImage';
import SmallCardDetail from './SmallCardDetail';
const SmallCardContainer = ({title,detail,cardImage})=>{
    return(
        <div className={style['card-without-container-container']}>
            <SmallCardImage cardImage={cardImage} />
            <SmallCardDetail title={title} detail={detail}/>
        </div>
    )
}
export default SmallCardContainer