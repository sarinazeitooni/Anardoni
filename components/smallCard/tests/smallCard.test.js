import React from 'react';
import { shallow } from 'enzyme';
import SmallCardContainer from "../SmallCardContainer";
describe("small card conatiner component" , ()=>{
   it("it should render a div",()=>{
       const container = shallow(<SmallCardContainer/>)
       expect(container).toMatchSnapshot();
   })
});