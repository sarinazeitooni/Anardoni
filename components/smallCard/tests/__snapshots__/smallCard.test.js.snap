// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`small card conatiner component it should render a div 1`] = `ShallowWrapper {}`;
